browser/test/Services/Explorer/ExplorerStoreTests.ts

```diff
@@ -38,8 +61,9 @@ describe("ExplorerStore", () => {
             })
 
             await TestHelpers.waitForAllAsyncOperations()
+            TestHelpers.tick(0) // execute setImmediate of memory-fs callbacks
+            await Promise.all(explorerFileSystem.promises)
```

Comment:
Awesome, thanks for adding an explicit way to wait for those operations to complete! 👍